templates/optimizers.yaml

```diff
@@ -41,53 +42,63 @@ spec:
 
-                if os.path.isfile('initial_parameters.json'):
-                  initial_parameters = load_circuit_template_params('initial_parameters.json')
+                if os.path.isfile("initial_parameters.json"):
+                    initial_parameters = load_circuit_template_params("initial_parameters.json")
                 else:
-                  initial_parameters = None
+                    initial_parameters = None
 
                 # Load qubit op
-                operator = load_qubit_operator('qubitop.json')
+                operator = load_qubit_operator("qubitop.json")
 
                 # Load parameter grid
-                if os.path.isfile('parameter_grid.json'):
-                  grid = load_parameter_grid('parameter_grid.json')
+                if os.path.isfile("parameter_grid.json"):
+                    grid = load_parameter_grid("parameter_grid.json")
                 else:
-                  grid = None
-                optimizer_specs = {{inputs.parameters.optimizer-specs}}
-                if grid is not None and optimizer_specs['function_name'] == 'GridSearchOptimizer':
-                  optimizer = create_object(optimizer_specs, grid=grid)
+                    grid = None
+                optimizer_specs = {{inputs.parameters.optimizer - specs}}
+                if grid is not None and optimizer_specs["function_name"] == "GridSearchOptimizer":
+                    optimizer = create_object(optimizer_specs, grid=grid)
                 else:
-                  optimizer = create_object(optimizer_specs)
-                  
-                backend_specs = {{inputs.parameters.backend-specs}}
-                if os.path.isfile('noise_model.json'):
-                  backend_specs["noise_model"] = load_noise_model("noise_model.json")
+                    optimizer = create_object(optimizer_specs)
+
+                backend_specs = {{inputs.parameters.backend - specs}}
+                if os.path.isfile("noise_model.json"):
+                    backend_specs["noise_model"] = load_noise_model("noise_model.json")
                 if os.path.isfile("device_connectivity.json"):
                     backend_specs["device_connectivity"] = load_circuit_connectivity(
                         "device_connectivity.json"
                     )
                 backend = create_object(backend_specs)
 
-                cost_function = EvaluateOperatorCostFunction(operator, ansatz, backend)
-                if os.path.isfile('constraint_operator.json'):
-                  constraint_operator = load_qubit_operator('constraint_operator.json')
-                  constraint_cost_function = EvaluateOperatorCostFunction(constraint_operator, ansatz, backend)
-                  constraint_functions = ({'type': 'eq', 'fun': constraint_cost_function.evaluate},)
-                  optimizer.constraints = constraint_functions
+                cost_function_specs = {{inputs.parameters.cost - function - specs}}
```

Comment:
Same here – it should be `{{inputs.parameters.cost-function-specs}}` not `{{inputs.parameters.cost - function - specs}}`.

Reply:
Perhaps it would be a good idea to revert all the other "reformatting" changes in this file as well, just to be safe.